python/dask_cudf/dask_cudf/io/parquet.py

```diff
@@ -316,7 +316,7 @@ def read_partition(
 
             if index and (index[0] in df.columns):
                 df = df.set_index(index[0])
-            elif index is False and df.index.names != (None,):
+            elif index is False and tuple(df.index.names) != (None,):
```

Comment:
Can you do direct comparison of a FrozenList and a list? That might be more “natural” if it works.
```suggestion
            elif index is False and df.index.names != [None]:
```